### PR TITLE
Specify that dart analyze supports single file analysis in Dart 2.13

### PR DESCRIPTION
**Pull request by parlough**
Fixes https://github.com/dart-lang/site-www/issues/3062

### CODE REVIEW

**Review by bwilkerson — APPROVED** [review]

File: src/tools/dart-analyze.md [hunk 1]
```diff
@@ -30,12 +30,22 @@ add the `--fatal-infos` flag:
 $ dart analyze --fatal-infos
 ```
 
-You can add a directory argument:
+You can add a directory or a single file argument:
+
+```terminal
+$ dart analyze [<DIRECTORY> | <DART FILE>]
```
Comment (kwalrath) [inline src/tools/dart-analyze.md:36]:
```suggestion
$ dart analyze [<DIRECTORY> | <DART_FILE>]
```

(Per https://developers.google.com/style/placeholders#explaining-placeholders)

File: src/tools/dart-analyze.md [hunk 2]
```diff
@@ -30,12 +30,22 @@ add the `--fatal-infos` flag:
...
+```
+
+For example, to analyze the `bin` directory:
 
 ```terminal
 $ dart analyze bin
 ```
 
+{{site.alert.version-note}}
+  Before Dart 2.13, only directory arguments were supported by the command.
```
Comment (kwalrath) [inline src/tools/dart-analyze.md:46]: Let's avoid the passive voice:

```suggestion
  Before Dart 2.13, `dart analyze` supported only directory arguments.
```

File: src/tools/dart-analyze.md [hunk 3]
```diff
@@ -30,12 +30,22 @@ add the `--fatal-infos` flag:
...
 ```
 
-You can add a directory argument:
+You can add a directory or a single file argument:
+
+```terminal
+$ dart analyze [<DIRECTORY> | <DART FILE>]
+```
+
+For example, to analyze the `bin` directory:
```
Comment (kwalrath) [inline src/tools/dart-analyze.md:39]:
```suggestion
For example, here's the command to analyze the `bin` directory:
```

(Per https://developers.google.com/style/code-samples?hl=en#introductory-statements)

**Review by kwalrath — APPROVED** [review]
LGTM, but I'd like a few tweaks

File: src/tools/dart-analyze.md (hunk 1 above)
Comment (parlough) [reply src/tools/dart-analyze.md:36]: There's really a section for everything, didn't notice this one. Thanks!

**Comment by kwalrath** [comment]
Thank you, @parlough!

**Pull request merged**